### PR TITLE
Fix for default item selection

### LINKED ISSUE #46: _scaleStart must be greater than 0
We're seeing a Javascript error when _scaleStart is set to 0 (like in the example.json).  This only happens when the user clicks submit _without first touching the slider handle_ with the handle set to 0.  This is fine if the user moves the handle to another number and back to 0 (or any other number), and then clicks submit.  adapt-contrib-slider does not seem to have this problem.

Suggest either allowing a 0 value or changing the example.json and plugin instructions.

**Javascript error**
sliderModel.js: this.get(...) is undefined

**Environment**
Adapt v5.7
Confidence Slider v3.0.0

### PR DESCRIPTION
**Pull request by chris-steele**
Fixes #46

### CODE REVIEW

**Comment by moloko** [comment]
@chris-steele nothing wrong with including the auto-formatting, just do as separate commit so it's easily separated out for reviewing

File: js/confidenceSliderModel.js [hunk 1]
```diff
@@ -41,9 +41,9 @@ define([
     restoreUserAnswers: function() {
       if (!this.get('_isSubmitted')) {
         this.set({
```
Comment (moloko) [inline js/confidenceSliderModel.js:43]: this could now be changed to the slightly-neater this.unset('_userAnswer');

**Review by moloko — APPROVED** [review]
certainly fixes the issue for me...

**Comment by moloko** [comment]
Just to note for whoever does the release of this that it will require a bump in the adapt-contrib-slider dependency in bower.json

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by eleanor-heath — APPROVED** [review]

**Pull request merged**